conan/ci_manager.py

```diff
@@ -92,7 +93,11 @@ def get_commit_msg(self):
 
     def get_branch(self):
         try:
-            msg = subprocess.check_output("git branch | grep \*", shell=True).decode().strip()
+            if platform.system() == "Windows":
```

Comment:
well, you could use `re` instead to invoke a pipe in the middle of command.

Reply:
Maybe it's easier to just read `.git/HEAD` and parse out the branch?

`ref: refs/heads/testing/0.1.0`

Reply:
I'm just afraid because is not secure invoke pipe on command. There is some better usage using POpen: https://docs.python.org/3.5/library/subprocess.html#replacing-shell-pipeline

However, as we "trust" in Travis, so I see no problem.

Reply:
Hi! I've opened a new PR #177. Please let me know if it could fix this.